server/auth/types/basic/routes.ts

```diff
@@ -25,6 +25,7 @@ import { SecurityClient } from '../../../backend/opensearch_security_client';
 import { API_AUTH_LOGIN, API_AUTH_LOGOUT, LOGIN_PAGE_URI } from '../../../../common';
 import { resolveTenant } from '../../../multitenancy/tenant_resolver';
 import { ParsedUrlQueryParams } from '../../../utils/next_url';
```

Comment:
Manual backport was required here because these imports differ slightly between main/2.x and 1.3 branch.